backend/src/core/usecase/AddTransactionToGroup.ts

```diff
@@ -47,9 +50,24 @@ export class AddTransactionToGroup implements IAddTransactionToGroup {
       };
     }
 
+    const recipients: GroupMember[] = [];
+    group.members.forEach((member) => {
+      if (recipientsId.includes(member.id)) {
+        recipients.push(member);
+      }
+    });
```

Comment:
```suggestion
    const recipients = group.members.filter(member => recipientsId.includes(member.id));
```

filter all the way! :fire: